x-pack/test_serverless/api_integration/services/svl_reporting.ts

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { X_ELASTIC_INTERNAL_ORIGIN_REQUEST } from '@kbn/core-http-common';
+import { INTERNAL_ROUTES } from '@kbn/reporting-plugin/common/constants';
+import expect from '@kbn/expect';
+import type { ReportingJobResponse } from '@kbn/reporting-plugin/server/types';
+import rison from '@kbn/rison';
+import { FtrProviderContext } from '../ftr_provider_context';
+
+const API_HEADER: [string, string] = ['kbn-xsrf', 'reporting'];
+const INTERNAL_HEADER: [string, string] = [X_ELASTIC_INTERNAL_ORIGIN_REQUEST, 'Kibana'];
+
+const DATA_ANALYST_PASSWORD = 'data_analyst-password';
+const DATA_ANALYST_ROLE = 'data_analyst_role';
+const DATA_ANALYST_USERNAME = 'data_analyst';
+const REPORTING_ROLE = 'reporting_user_role';
+const REPORTING_USER_PASSWORD = 'reporting_user-password';
+const REPORTING_USER_USERNAME = 'reporting_user';
+
```

Comment:
@rshen91 in serverless tests, for the foreseeable future, it looks like we won't be able to run tests that use custom roles or users. It looks like we need to rely on just the `elastic_serverless` user:  that's the user that it automatically logged in the beginning of every functional test.

I'm not sure how we can test that users can only see their own reports, but that ability might come later to the serverless tests configuration.